src/resourcemanager/resourcespace/events.py

```diff
@@ -36,8 +36,17 @@ def upload_image(obj, event):
     registry = api.portal.get_tool('portal_registry')
     reg_prefix = 'resourcemanager.resourcespace.settings.IResourceSpaceKeys'
     upload_to_rs = registry['{0}.upload_to_rs'.format(reg_prefix)]
+    
+    #check each upload instance before uploading to RS
+    upload_this_to_rs = getattr(obj, 'upload_this_to_rs', None)
```

Comment:
upload_this_to_rs = getattr(obj, 'upload_this_to_rs', True)

Then you don't have to check for None below